.circleci/config.yml

```diff
@@ -3,10 +3,10 @@ jobs:
   j8_jvm_upgrade_dtests:
     docker:
     - image: spod/cassandra-testing-ubuntu1810-java11-w-dependencies:20190306
-    resource_class: medium
```

Comment:
I assume you didn't intend to commit this...

Reply:
right, not to commit. it's to speed up CI during our circle-ci trial period..